setup.py

```diff
@@ -15,14 +15,24 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import codecs
 import os
+import re
 
 from setuptools import setup
 
+# look/set what version we have
+changelog = 'debian/changelog'
+if os.path.exists(changelog):
+    head = codecs.open(changelog, encoding='utf-8').readline()
+    match = re.compile('.*\((.*)\).*').match(head)
+    if match:
+        version = match.group(1)
```

Comment:
I know the possibility of this happening is small, but should `version` default to "devel" so if this fails there's still a valid version?